immuni_exposure_ingestion/models/swagger.py

```diff
@@ -89,6 +89,9 @@ class Upload:
     """
 
     province = doc.String(description="The users's Province of Domicile.", required=True)
+    country_of_interest = doc.List(
+        str, description="The users's visited countries.", required=False
```

Comment:
If this is the description, why calling it `of interest` when `visited_countries` would be a way more descriptive name?

Reply:
our mistake! it should be countries_of_interest as stated in the ingestion class